scripts/setup

```diff
@@ -20,8 +20,6 @@ then
             usage
             ;;
         *)
-            docker-compose up -d database
-            ./scripts/migrate
```

Comment:
`scripts/setup` job is to prepare the local development environment so that `scripts/update` can be applied. There's nothing that needs to be done after a repository clone (e.g. a Vagrant provision) so this file shouldn't do anything besides invoke `scripts/update`.